rtloader/test/python/datadog_checks/base/checks/__init__.py

```diff
@@ -1,5 +1,5 @@
 # AgentCheck stubs for testing
-class AgentCheck(object):
+class AgentCheck(object):  # noqa: UP004
```

Comment:
is this inheritance useless? Or is it still used for python2?

Reply:
Still there for py2 tests